benchmark/coinmarketcap/coinmarketcap.com.bench.js

```diff
@@ -0,0 +1,100 @@
+import assert from 'assert';
+import { fox } from '../../src/index.js';
+import { objectCartesian, benchmarkOptions } from '../lib.js';
+import { matchesPatterns, runCrawlBenchmark } from '../lib/crawl.js'
+
+const coinmarketEval = {
+  includePatterns: new RegExp('^https://coinmarketcap.com/currencies/'),
+  //includePatterns: new RegExp('currencies/'),
+}
+
+const coinmarketOptions = {
+  url: 'https://ffcloud.s3.us-west-2.amazonaws.com/testdata/coinmarketcap.html',
+  prompt: [
+    'find lnks to pages about coins, and get their basic market data, including 24 hour trading volume',
+    'get cryptocurrencies',
+    'links to crypto',
```

Comment:
1) These prompts should be used one-by-one, not all together
2) They should be passed in to crawl(), so like this:

```
for (const prompt of prompts) {
    const out = await fox.init(url).crawl(prompt);
}
```